src/Stratis.Bitcoin.Features.Apps/AppsStore.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.Apps.Interfaces;
+using Stratis.Bitcoin.Utilities;
+using Stratis.Bitcoin.Utilities.Extensions;
+
+namespace Stratis.Bitcoin.Features.Apps
+{
+    public class AppsStore : IAppsStore
+    {
+        private readonly ILogger logger;
+        private readonly List<IStratisApp> applications = new List<IStratisApp>();
+        private readonly DataFolder dataFolder;
+        private const string ConfigFileName = "stratisApp.json";
+
+        public AppsStore(ILoggerFactory loggerFactory, DataFolder dataFolder)
+        {
+            this.dataFolder = dataFolder;
+            this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
+
+            this.Load();
+        }
+
+        public IEnumerable<IStratisApp> Applications => this.applications;
+
+        private void Load()
+        {
+            try
+            {
+                FileInfo[] fileInfos = new DirectoryInfo(this.dataFolder.ApplicationsPath)
```

Comment:
We probably want to look in a specific folder no?

Reply:
Nope - this code is simply looking for all stratisApp.json files under the apps folder